vtex-sites/gatsby.store
# chore: Organize some search components

## What's the purpose of this pull request?

It's to address some of the visual inconsistencies in the search components `History`, `SuggestionsTopSearch`, and `Suggestions`.

## How does it work?

Fix clickable areas, unnecessary duplicated CSS, padding, fonts, etc.

Before|After
-|-
![History-1](https://user-images.githubusercontent.com/381395/165192183-5d1527ce-b035-43e6-86bb-a3a6655f9d49.gif)|![History-2](https://user-images.githubusercontent.com/381395/165192194-670a9c43-0e00-41d8-af35-b1abde30bcd4.gif)
![TopSearch-1](https://user-images.githubusercontent.com/381395/165192213-85f43d4d-a094-4315-aa67-d178c5e00db7.gif)|![TopSearch-2](https://user-images.githubusercontent.com/381395/165192217-1f578a23-d653-452c-b8d5-409e38df43b2.gif)
![Suggestions-1](https://user-images.githubusercontent.com/381395/165192232-634a1c8f-18f6-40e2-b96d-d3c638689020.gif)|![Suggestions-2](https://user-images.githubusercontent.com/381395/165192237-5525590c-9658-4ed6-8b46-8d5be18bac40.gif)

## How to test it?

Check the stories for these components [here](https://gatsby-store-git-refactor-search-history-n-top-faststore.vercel.app/?path=/story/organisms-search-history--default).
